GameView.py

```diff
@@ -37,11 +30,19 @@
 from ConfigParser import SafeConfigParser
 from Globales import COLORES
 
+import gi
+from gi.repository import Gtk, GObject, Pango
+gi.require_version('Gtk', '3.0')
+
+installed_dir = os.path.abspath(os.path.dirname(__file__))
+sys.path.insert(1, os.path.join(installed_dir, "Lib/"))
```

Comment:
By moving the change to `sys.path` you prevent the following imports from using the adjusted path.